### ISSUE TITLE
2 erros que fica após atualização do gimbal do dji mini2

### ISSUE THREAD

**Issue by LeandroBrasill** [o-gs/dji-firmware-tools#201]
[https://github.com/o-gs/dji-firmware-tools/issues/186#issuecomment-771756867]() 
 
Alguém consegue ou já conseguiu tirar esse 2 erros???

![Screenshot_2021-02-01-22-53-59-127_dji go v5](https://user-images.githubusercontent.com/77772685/115090975-a5704180-9eec-11eb-9a59-e21cd4736468.jpg)

**Comment by jefsilv** [comment]
YEs, here in Brazil, there is just a guy that do this, HD Drones his name is Robson, and cost around R$ 350,00 ... would´like learn how do

**Comment by 954426470** [comment]
I will, you can contact me
